src/test/kotlin/net/atos/zac/app/zaken/ZakenRESTServiceTest.kt

```diff
@@ -122,6 +123,7 @@ class ZakenRESTServiceTest : BehaviorSpec({
     val zgwApiService: ZGWApiService = mockk<ZGWApiService>()
     val zrcClientService: ZRCClientService = mockk<ZRCClientService>()
     val ztcClientService: ZtcClientService = mockk<ZtcClientService>()
+    val restZaakHistorieRegelConverter = mockk<RESTZaakHistorieRegelConverter>()
```

Comment:
But no actual test that uses it?

Reply:
Added a few 2 unit test cases and an integration test.
Please have a look